mangaki/mangaki/models.py

```diff
@@ -229,31 +230,57 @@ def __str__(self):
 
 class WorkTitle(models.Model):
     work = models.ForeignKey('Work')
-    title = models.CharField(max_length=128, blank=True, db_index=True)
+    # 255 should be safe, we have seen titles of 187 characters in Japanese.
+    # So we could expect longer titles in English.
+    title = models.CharField(max_length=255, blank=True, db_index=True)
     title_search = SearchVectorField('title')
     language = models.ForeignKey('Language',
                                  null=True)
+    ext_language = models.ForeignKey('ExtLanguage',
+                                     null=True)
     type = models.CharField(max_length=9, choices=(
                             ('main', 'principal'),
                             ('official', 'officiel'),
-                            ('synonym', 'synonyme')),
+                            ('synonym', 'synonyme'),
+                            ('short', 'court')),
                             blank=True,
                             db_index=True)
 
     def __str__(self):
-        return "%s" % self.title
+        return ("{} - {} (source: {}, type: {}) attached to {}"
+                .format(self.title, self.language.code, self.ext_language.source, self.type, self.work))
```

Comment:
language = ext_language = NULL → we could default language to None (unknown language).